packages/@aws-cdk/aws-glue-alpha/lib/job-executable.ts

```diff
@@ -253,7 +269,7 @@ export interface PythonShellExecutableProps extends SharedJobExecutableProps, Py
 /**
  * Props for creating a Python Ray job executable
  */
-export interface PythonRayExecutableProps extends SharedJobExecutableProps, PythonExecutableProps {}
+export interface PythonRayExecutableProps extends SharedJobExecutableProps, RayExecutableProps {}
```

Comment:
what happens to `PythonExecutableProps`? a) is it used elsewhere and b) we are not dropping support for any properties right?

Reply:
if we are dropping support for properties (as the attached issue suggests) it should have been called out in PR description as a `BREAKING CHANGE`.

Reply:
a) is it used elsewhere
Yes. There are three job types; Spark, Python shell, and Ray. PythonExecutableProps is used for the first two.

 b) we are not dropping support for any properties right?
Right. We are not dropping any or working properties.
Only the property we are dropping is `extraPythonFiles` which was not working for Ray job. This is not breaking change.